collections/ansible_collections/purestorage/flasharray/plugins/modules/purefa_pg.py

```diff
@@ -730,7 +730,6 @@ def main():
     array = get_system(module)
     pattern = re.compile("^[a-zA-Z0-9]([a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?$")
     if module.params["rename"]:
-        module.params["rename"] = module.params["rename"].lower()
```

Comment:
We can't just ignore the case as Purity is case insensitive.
Given Ansible still supports Python2.7 we have to force lowercase for consistency across all the modules.
When Python 2.7 is deprecated by Ansible there are a number of modules that will have `lower()` replaced with `casefold()` to allow case insensitivity to be fully supported by the Pure modules

Reply:
I see your point. On the other hand the UI allows creation and renaming in mixed case.
Maybe the introduction of a "lowercase_names" (bool, default True) option could be a solution to this?

Reply:
There are many modules that would need this if we are to be consistent. In the interim we will need to stick to lowercase

Reply:
Removing this change pending Python 2.7 deprecation